apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/doedshendelse/DoedshendelseJobService.kt

```diff
@@ -100,6 +104,19 @@ class DoedshendelseJobService(
         }
     }
 
+    private fun sendBrevHvisKravOppfyllesBP(
```

Comment:
Er det egentlig noen grunn til å ha denne BP-spesifikk?

Reply:
Ikke i lengden men for now håndterer vi bare barn

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/3946/commits/b656e22cc27948123397ea13a045a2ddd68ee028